### PR TITLE
Fixes Quartermaster's office on other stations that doesn't have an Announcement Request console

### LINKED ISSUE #22339: The QM office on the Cyberiad doesn't have an announcement console
**Issue Description**:
There is no console to make announcements with in the QM office

**What did you expect to happen**: 
To announce the glory of cargonia via announcement.

**What happened instead**:
Had to walk to the bridge :(

**Why is this bad/What are the consequences**:
Every head office has one, QM is a head now

**When did the problem start happening**:
Exactly 5 minutes ago

### PR DESCRIPTION
**Pull request by Fluffyzack21**
## What Does This PR Do
 It allows the Quartermaster to announce things in there office

 Fixes #22339 

## Why It's Good For The Game
 The Quartermaster can now announce to the crew about his issues

## Images of changes
 
Screenshot 2023-12-01 210818

## Testing
 Spawned as the Quartermaster and made announcement and it does work on NSS Cyberiad

## Changelog
:cl:
Fix: You now can announce stuff as the Quartermaster in the Quartermaster's office on the Requests Console on NSS Cyberiad, kerberos, ceberon, farragus,
/:cl:

### CODE REVIEW

**Comment by datlo** [comment]
While you're remapping QM office, can you look at the ID authentificator for red alert/ERT ? IIRC it's buried under stuff on the table, would be nice if it wasn't

**Comment by Sadhorizon** [comment]
Other stations also do not have it set up. As a matter of fact, only mining outpost does currently.

**Comment by Fluffyzack21** [comment]
> While you're remapping QM office, can you look at the ID authentificator for red alert/ERT ? IIRC it's buried under stuff on the table, would be nice if it wasn't

Screenshot 2023-12-02 074355

**Comment by Fluffyzack21** [comment]
Screenshot 2023-12-02 075948

**Comment by Fluffyzack21** [comment]
Screenshot 2023-12-02 081039
Screenshot 2023-12-02 080408
Screenshot 2023-12-02 075948

**Comment by Fluffyzack21** [comment]
> Other stations also do not have it set up. As a matter of fact, only mining outpost does currently.

The only station that has one currently is farrargus, is which why I made this PR https://github.com/ParadiseSS13/Paradise/pull/23448 this will fix the issues

**Comment by Sadhorizon** [comment]
> > Other stations also do not have it set up. As a matter of fact, only mining outpost does currently.
> 
> The only station that has one currently is farrargus, is which why I made this PR #23448 this will fix the issues

I was referring to request consoles.

**Comment by Fluffyzack21** [comment]
> > Other stations also do not have it set up. As a matter of fact, only mining outpost does currently.
> 
> The only station that has one currently is farrargus, is which why I made this PR #23448 this will fix the issues

Ah I fixed it then

**Comment by Fluffyzack21** [comment]
> Update your PR description since you are touching another thing too on Cyberiad

fixed

**Comment by Fluffyzack21** [comment]
Screenshot 2023-12-08 082931

File: _maps/map_files/cyberiad/cyberiad.dmm [hunk 1]
```diff
@@ -22740,12 +22740,14 @@
...
 /obj/structure/table/reinforced,
 /obj/item/book/manual/wiki/sop_command,
-/obj/item/aicard,
-/obj/item/multitool/command,
 /obj/machinery/keycard_auth{
 	pixel_x = -6;
 	pixel_y = 8
 	},
+/obj/item/aicard{
+	pixel_x = null
```
Comment (warriorstar-orion) [inline _maps/map_files/cyberiad/cyberiad.dmm:22748]: Weird varedit happened here

File: _maps/map_files/cerestation/cerestation.dmm [hunk 2]
```diff
@@ -24861,7 +24861,8 @@
 	department = "Cargo Bay";
 	departmentType = 2;
```
Comment (warriorstar-orion) [inline _maps/map_files/cerestation/cerestation.dmm:24862]: this departmentType should also be 5 I think?

**Review by warriorstar-orion — CHANGES_REQUESTED** [review]

File: _maps/map_files/cyberiad/cyberiad.dmm (hunk 1 above)
Comment (Fluffyzack21) [reply _maps/map_files/cyberiad/cyberiad.dmm:22748]: Yeah I don't know either what happened but I fixed it

File: _maps/map_files/cerestation/cerestation.dmm (hunk 2 above)
Comment (Fluffyzack21) [reply _maps/map_files/cerestation/cerestation.dmm:24862]: I fixed it, its now set to 5

**Review by warriorstar-orion — APPROVED** [review]
Thanks for the changes, LGTM

**Review by Aylong220 — APPROVED** [review]

**Review by Sirryan2002 — APPROVED** [review]

**Review by lewcc — APPROVED** [review]
Looks good to go once MQL catches up

**Pull request merged**